packages/react-native/template/ios/HelloWorld/main.swift

```diff
@@ -0,0 +1,8 @@
+import UIKit
+
+UIApplicationMain(
```

Comment:
I don't think this is needed, you can annotate `AppDelegate` with `@main`. Check newly generated project from Xcode

Reply:
Resolved under 92a1ae0. I wanted to keep the file structures the same for review, but if we are committing to use Swift, might as well take advantage of it. And `@main` was introduced in 5.3, so pretty much everyone should have it.